packages/salesforcedx-vscode-lwc-next/package.json

```diff
@@ -25,6 +25,8 @@
   "dependencies": {
     "@salesforce/salesforcedx-utils-vscode": "44.8.0",
     "ajv": "^6.1.1",
+    "eslint": "^4.19.1",
+    "eslint-plugin-lwc": "^0.5.1",
```

Comment:
We need to define a tagging strategy and avoid specifying specific versions. See W-5625040.

Reply:
@kevinv11n - I looked at W-5625040 and it says "Bundle eslint and @salesforce/eslint-plugin-lwc with in the LWC extension". 

Do you just want to update the dependency so that it's `@salesforce/eslint-plugin-lwc` instead of `eslint-plugin-lwc` and we can handle the version number later?